Source/Krypton Components/Krypton.Toolkit/Controls Toolkit/KryptonFolderBrowserDialog.cs

```diff
@@ -62,9 +62,9 @@ public class KryptonFolderBrowserDialog : ShellDialogWrapper, IDisposable
         [Category(@"FolderBrowsing")]
         [Description(@"Sets the directory path of the initial folder shown in the dialog box.")]
         [AllowNull]
-        public string SelectedPath
+        public string? SelectedPath
```

Comment:
`[AllowNull]` means that it can accept a null, but will not return a null.
So the type should be `string` only

Reply:
Reverted